docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,97 @@
+.. _enable_intel_gpu:
+
+Enable Intel GPUs
+=============================
+
+This guide describes how to configure DSS to utilise the Intel GPUs on your machine. This can be done by enabling the Intel device plugin on your MicroK8s cluster.
+
+Prerequisites
+-------------
+
+* :ref:`MicroK8s is installed <set_microk8s>`.
+* :ref:`DSS CLI is installed <install_DSS_CLI>`.
+* The Intel GPU is recognised by your machine. This can be confirmed by running `intel_gpu_top` from the `intel-gpu-tools` package. For discrete GPUs on older versions of Ubuntu like 22.04, you may need to perform additional steps like install a HWE kernel.
+
+Install the kubectl snap
+---------------------------
+First, install the `kubectl` snap package:
+
+.. code-block:: bash
+				
+  sudo snap install kubectl -–classic
+
+Create a configuration file so that `kubectl` can communicate with MicroK8s by running:
+
+.. code-block:: bash
+				
+  mkdir -p ~/.kube
+  microk8s config > ~/.kube/config
+
+Enable the Intel GPU plugin in your Kubernetes cluster 
+------------------------------------------------------
+
+To ensure DSS can utilise Intel GPUs, you have to enable the Intel GPU plugin in your Kubernetes cluster. The plugin uses the `Intel device plugins for Kubernetes`_ to detect whether an Intel GPU exists on a machine. You can see the `Installation with NFD`_ page for more details.
+
+To enable the plugin, you must download the appropriate `kustomization.yaml` files from Intel’s device plugins for Kubernetes repository, and then apply them on your MicroK8s cluster.
```

Comment:
```suggestion
To enable the plugin, download the appropriate `kustomization.yaml` files from Intel’s device plugins for Kubernetes repository, and then apply them on your MicroK8s cluster.
```

Reply:
Add link to the repository here. I assume it is the same one as above, then remove it from there (probably the whole sentence), and add it here. Users will likely check it at this point and not before.

Reply:
Done